src/test/java/org/opentripplanner/transit/raptor/speed_test/model/testcase/ItineraryResultMapper.java

```diff
@@ -130,7 +130,7 @@ private Result map(Itinerary itinerary) {
     return new Result(
```

Comment:
Should we modify Result to contain `Duration`?

Reply:
Done